src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -108,4 +107,13 @@ export const Checkmark = styled.div`
   width: 18px;
   height: 18px;
   border-radius: 1px;
+  position: relative;
+  svg {
+    position: absolute;
+  }
+`
+
+export const StyledHelperText = styled(HelperText)`
+  margin-top: 0;
```

Comment:
```suggestion
```

Reply:
I set it to `sizes(1)` eventually, in the design margin is a little bit different for TextField and Checkbox